config/config.go

```diff
@@ -36,10 +36,10 @@ type StorageConfig struct {
 	Bloom BloomFilterConfig `json:"bloom"`
 }
 
-// AddressConfig will map the json address configuration
-type AddressConfig struct {
+// PubkeyConfig will map the json public key configuration
```

Comment:
Not quite, it works with public keys (string <-> bytes) and thus, that is the name PubkeyConverter. Note that it works on both types of keys (BLS + tx sign) OR, it can work with any arbitrary set of bytes (could have been hashes, marshalized stuff, anything that can be represented as bytes)